<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>40</y>
      <width>101</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="ButtonShot">
       <property name="text">
        <string>单次采集</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOpen">
       <property name="text">
        <string>打开相机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonStop">
       <property name="text">
        <string>停止采集</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Property">
       <property name="text">
        <string>属性</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonSave">
       <property name="text">
        <string>保存图像</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1198</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
